### PR TITLE
[Clarity] Clarity unit tests for sBTC Mini token contract

### PR DESCRIPTION
**Pull request by MarvinJanssen**
Unit tests for the sBTC Mini token contract. The tests are written in Clarity and uses the script of #269 to generate Clarinet test stubs.

### CODE REVIEW

**Comment by MarvinJanssen** [comment]
Github Action blocked by https://github.com/hirosystems/clarinet/issues/975.

**Comment by jo-tm** [comment]
@MarvinJanssen this could become a Grant.

**Comment by jo-tm** [comment]
We need the Binary of Clarinet for Ubuntu Latest (Jose).

File: .github/workflows/clarinet.yml [hunk 1]
```diff
@@ -26,10 +26,20 @@ jobs:
...
-      - name: Run clarinet test
+      - name: Run clarinet test for sbtc-ops
         working-directory: ./sbtc-ops/clarinet
-        run: clarinet test
\ No newline at end of file
+        run: clarinet test
+
+      - name: Run unit tests for sbtc-mini
+        working-directory: ./sbtc-mini
+        run: ./scripts/test.sh
```
[the remark this answers is not in the document]
Comment (jo-tm) [reply .github/workflows/clarinet.yml:39]: solved.

**Review by jo-tm — APPROVED** [review]

**Comment by jo-tm** [comment]
Tests for some the contracts, more tests in a later PR.

File: bin/install_clarinet_ubuntu.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+apt-get update
+apt-get install -y unzip curl
+
+curl -L -o clarinet 'https://drive.google.com/uc?export=download&confirm=yes&id=1amZ-VC53P8A2NAnwQ2bcduNBrzC-EzG7'
```
Comment (jo-tm) [inline bin/install_clarinet_ubuntu.sh:6]: This is a patch @sergey-shandar, we can build or host the version we need of `clarinet` wherever you want.

Comment (jo-tm) [reply bin/install_clarinet_ubuntu.sh:6]: I mean later on.

**Comment by MarvinJanssen** [comment]
That's awesome @jo-tm! Is codecov no longer showing up?

**Review by FriendsFerdinand — APPROVED** [review]

File: bin/install_clarinet_ubuntu.sh (hunk 2 above)
Comment (MarvinJanssen) [reply bin/install_clarinet_ubuntu.sh:6]: Until the release, this is a good enough workaround.

**Pull request merged**